2018-11/index.md

```diff
@@ -4,69 +4,69 @@ title: BeagleBoard.org Enables Success
-   body: Check out with Prof Yoder’s course with materials posted in this thread.
- - title: Qt5 "hello world" how-to on BeagleBone Debian
-   link: https://groups.google.com/forum/embed/?place=forum/beaglebone&showsearch=true&showpopout=true&showtabs=false&hideforumtitle=true&parenturl=https%3A%2F%2Fbeagleboard.org%2Fdiscuss%2F%3Fplace%3Dtopic%2Fbeagleboard%2FoMCYbQgzjQo%2Fdiscussion#!category-topic/beagleboard/software/bGuAcHm0atc
-   body: Need to use Qt on an embedded system – check this post for latest getting started
+ - title: Fresh Debian 9.5 Images
+   link: https://beagleboard.org/blog/2018-10-10-fresh-debian-9-5-images
+   body: Download a new image for fixes for zero-install drivers on Windows 10 and much more!
+ - title: Getting Started with PocketBeagle® Workshop Materials
+   link: https://beagleboard.org/workshop
+   body: Maker Faire Education Day workshop image, examples and slides are now available.
+ - title: Harness the Performance of the PRUs
+   link: http://beagleboard.org/pru
+   body: The programmable real-time units (PRUs) are fast 32-bit processors within the AM3358 with single-cycle I/O access.
 projects:
- - title: Bela Mini - Low-Latency Audio + Sensor Cape for PocketBeagle®
-   link: http://bbb.io/+1615e0
-   body: Bela Mini creates interactive audio projects like musical instruments, sound installations & assistive technology.
-   image: https://beagleboard.github.io/newsletter/static/images/newsletter-2018-09_0004.jpg
- - title: Bipolar Stepper Motor with BeagleBone® Blue
-   link: http://bbb.io/+e4950e
-   body: BeagleBone® Blue's DC motor drivers can be used to drive bipolar stepper motors too! Pair them up with a little code for open loop control.
-   image: https://beagleboard.github.io/newsletter/static/images/newsletter-2018-09_0005.JPG
- - title: PocketBeagle® Automatic Light Switch
-   link: http://bbb.io/+f0588f
-   body: Beginner project creates an automatic light switch using a simple servo, PocketBeagle® and PIR motion sensor!
-   image: https://beagleboard.github.io/newsletter/static/images/newsletter-2018-09_0006.PNG
- - title: Smart Mirror
-   link: http://bbb.io/+559755
-   body: Students at University of Texas built their own custom mirror with BeagleBone® Black and parts from the hardware store.
-   image: https://beagleboard.github.io/newsletter/static/images/newsletter-2018-09_0007.JPG
+ - title: SCUTTLE Mobile Utility Robot 
+   link: https://beagleboard.org/p/112547/scuttle-mobile-utility-robot-from-texas-a-m-d6c2b7
+   body: Educational Platform from Texas A&M Mechatronics Engineering Technology Program using BeagleBone® Blue
+   image: https://hackster.imgix.net/uploads/attachments/612564/img_e2371_smB8gEhktG.JPG
+ - title: BeagleBone® Green Wireless Irrigation Control
+   link: https://beagleboard.org/p/Greg-R/beaglebone-green-wireless-irrigation-control-ce7c4b
```

Comment:
```suggestion
   link: https://bbb.io/+ce7c4b
```

You don't HAVE to use these short links, but I think they are kinda handy and nice to promote. The format is https://bbb.io/+ followed by the last 6 characters in the project link.

Reply:
gotcha